docs/spire.md

```diff
@@ -0,0 +1,284 @@
+            -selector k8s:sa:tekton-pipelines-controller
+    
+    ```
+1. Modify the controller (`config/controller.yaml`) to provide access to the SPIRE agent socket.
+    ```yaml
+    # Add the following the volumeMounts of the "tekton-pipelines-controller" container
+    - name: spiffe-workload-api
+      mountPath: /spiffe-workload-api
+      readOnly: true
+    
+    # Add the following to the volumes of the controller pod
+    - name: spiffe-workload-api
+      csi:
+        driver: "csi.spiffe.io"
+    ```
+1. (Optional) Modify the controller (`config/controller.yaml`) to configure non-default SPIRE options by adding arguments to the CLI.
+    ```yaml
+          containers:
+          - name: tekton-pipelines-controller
+            image: ko://github.com/tektoncd/pipeline/cmd/controller
+            args: [
+              # These images are built on-demand by `ko resolve` and are replaced
+              # by image references by digest.
+              "-kubeconfig-writer-image", "ko://github.com/tektoncd/pipeline/cmd/kubeconfigwriter",
+              "-git-image", "ko://github.com/tektoncd/pipeline/cmd/git-init",
+              "-entrypoint-image", "ko://github.com/tektoncd/pipeline/cmd/entrypoint",
+              "-nop-image", "ko://github.com/tektoncd/pipeline/cmd/nop",
+              "-imagedigest-exporter-image", "ko://github.com/tektoncd/pipeline/cmd/imagedigestexporter",
+              "-pr-image", "ko://github.com/tektoncd/pipeline/cmd/pullrequest-init",
+              "-workingdirinit-image", "ko://github.com/tektoncd/pipeline/cmd/workingdirinit",
+    
+              # Configure optional SPIRE arguments
+    +         "-spire-trust-domain", "example.org",
+    +         "-spire-socket-path", "/spiffe-workload-api/spire-agent.sock",
+    +         "spire-server-addr", "spire-server.spire.svc.cluster.local:8081"
+    +         "spire-node-alias-prefix", "/tekton-node/",
+    
+              # This is gcr.io/google.com/cloudsdktool/cloud-sdk:302.0.0-slim
+              "-gsutil-image", "gcr.io/google.com/cloudsdktool/cloud-sdk@sha256:27b2c22bf259d9bc1a291e99c63791ba0c27a04d2db0a43241ba0f1f20f4067f",
+              # The shell image must be root in order to create directories and copy files to PVCs.
+              # gcr.io/distroless/base:debug as of October 21, 2021
+              # image shall not contains tag, so it will be supported on a runtime like cri-o
+              "-shell-image", "gcr.io/distroless/base@sha256:cfdc553400d41b47fd231b028403469811fcdbc0e69d66ea8030c5a0b5fbac2b",
+              # for script mode to work with windows we need a powershell image
+              # pinning to nanoserver tag as of July 15 2021
+              "-shell-image-win", "mcr.microsoft.com/powershell:nanoserver@sha256:b6d5ff841b78bdf2dfed7550000fd4f3437385b8fa686ec0f010be24777654d6",
+            ]
+    ```
+
+## Sample TaskRun attestation
+
+To demonstrate, we will use a simple task run that writes some results:
```

Comment:
It's hard to parse the intention of this sentence. I suggest a more direct approach here

"The following example shows how this feature works:"